Examples/options/geant_userLimits.py

```diff
@@ -0,0 +1,60 @@
+from Gaudi.Configuration import *
+
+# Data service
+from Configurables import FCCDataSvc
+podioevent = FCCDataSvc("EventDataSvc")
+
+from Configurables import HepMCFileReader, GenAlg
+readertool = HepMCFileReader("ReaderTool", Filename="/eos/project/f/fccsw-web/testsamples/FCC_minbias_100TeV.dat")
+reader = GenAlg("Reader", SignalProvider=readertool)
+reader.hepmc.Path = "hepmc"
+
+# reads an HepMC::GenEvent from the data service and writes a collection of EDM Particles
+from Configurables import HepMCToEDMConverter
+hepmc_converter = HepMCToEDMConverter("Converter")
+hepmc_converter.hepmc.Path="hepmc"
+hepmc_converter.genparticles.Path="allGenParticles"
+hepmc_converter.genvertices.Path="allGenVertices"
+
+# DD4hep geometry service
+# Parses the given xml file
+from Configurables import GeoSvc
+geoservice = GeoSvc("GeoSvc", detectors=['file:Detector/DetFCChhBaseline1/compact/FCChh_DectEmptyMaster.xml',
+                                         'file:Detector/DetFCChhBaseline1/compact/FCChh_TrackerAir.xml'],
+                                         OutputLevel=INFO)
+
+# Geant4 service
+# Configures the Geant simulation: geometry, physics list and user actions
+from Configurables import SimG4Svc, SimG4UserLimitPhysicsList, SimG4UserLimitRegion
+from GaudiKernel.SystemOfUnits import mm
+## create region and a parametrisation model, pass smearing tool
+regiontool = SimG4UserLimitRegion("limits", volumeNames=["TrackerEnvelopeBarrel"],
+                                       maxStep = 1)
```

Comment:
Could you add unit ? I think you planned to, judging from import two lines above.